ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -74,11 +88,14 @@ classes:
 
     LogicalVolume:
         label: Logical Volume
-        impacts:
-            - snapshotVolumes
-            - openstack_core_components
-        impacted_by:
-            - volumeGroup
+        monitoring_templates: ['LogicalVolume']
+        dynamicview_views: [service_view]
+        dynamicview_relations:
+            impacts:
```

Comment:
The FileSystem impact adapter creates a relationship between FileSystem and its LogicalVolume. The reverse of that needs to be defined here.